ec2-config/instance.tf

```
@@ -0,0 +1,93 @@
+# Configure the S3 backend
+terraform {
+  backend "s3" {
+      bucket = "terraform-state-bucket-180122"
+      key = "global/s3/ec2/terraform.tfstate"
```

It's better to have similar folder names in your repositories and in s3, otherwise it will become confusing when we have more layers
```suggestion
      key = "ec2-config/terraform.tfstate"
```
Please destroy ec2-config before changing this parameter